moq-pub/src/main.rs

```diff
@@ -41,7 +43,10 @@ async fn main() -> anyhow::Result<()> {
 	join_set.spawn(async move { log_viewer.run().await.context("failed to run media source") });
 
 	// TODO: generate unique namespace with UUID and/or take a command line arg
```

Comment:
We can check this off now :)

Reply:
I realize now that I had introduced that second hardcoded namespace string in `src/media.rs` after leaving this comment so it ended up being a bit misleading, sorry!

Reply:
I missed this one :). Removed it.